.gitignore

```diff
@@ -37,4 +37,7 @@ functions/.wrangler
 .wrangler/
 src/data/flood_control/lookups/contractors
 contractor_fetch_log.csv
-wrangler.toml
\ No newline at end of file
+wrangler.toml
```

Comment:
Please don't to prevent accidental push

Reply:
Need to be confident that our wrangler.toml is secured first before committing it to the repo. Until then please keep it in .gitignore.

Reply:
@jasontorres the diff shows the `wrangler.toml` line as being removed AFAIK because a newline was added to the line. `wrangler.toml` is still there.

the way i understand it, the original line is `wrangler.toml` and the new line is `wrangler.toml\n` which is why they are shown as being different even though in the diff there is no visible difference.